src/components/Spotify.js

```diff
@@ -1,388 +1,48 @@
 import React from 'react' 
 import SpotifyWebApi from 'spotify-web-api-js'
 const spotifyApi = new SpotifyWebApi()
+import SpotifyCtrl from 'spotify_ctrl'
 
+/* MSB: This is the spotify component; it simply renders the album art
+ * for the current playlist.  Most of the state and API calls are in 
+ * spotify_ctrl.js
+ */
 export default class Spotify extends React.Component{
   constructor(props){
     super(props)
-    const params = this.getHashParams()
-    const token = params.access_token
-    if(token){
-      spotifyApi.setAccessToken(token)
-    }
+    this.spotifyCtrl = new SpotifyCtrl()
```

Comment:
Line 13 should be in App.js or Genius.js. Instance of spotify control passed down